docs_user/modules/proc_migrating-mon-from-controller-nodes-verification.adoc

```diff
@@ -0,0 +1,16 @@
+.Verification
+
+* Verify that the {CephCluster} cluster is healthy:
++
+----
+[ceph: root@oc0-controller-0 specs]# ceph -s
+  cluster:
+    id:     f6ec3ebe-26f7-56c8-985d-eb974e8e08e3
+    health: HEALTH_OK
+...
+...
+----
+
+.Next steps
+
+* Repeat this procedure for any additional Controller node that hosts a Ceph Monitor until you migrate all the Ceph Monitor daemons to the target nodes.
```

Comment:
I'm thinking we could merge this verification step with the "Redeploying" procedure, since that's the last procedure they need to complete. 
And the "Next steps" could include links to the procedures they just did. 

I will make this update in the next iteration.

Reply:
ack, the intent of having the `-verification` procedure is because we might need to eventually expand this part. In that case, instead of growing the `Redeploy` part, we can simply add more steps here.
I already have in mind post actions, but considering I still don't have a draft I'm ok to remove this `-verification.adoc` and merge w/ the redeploy procedure.